sdk/maps/azure-maps-render/src/main/java/com/azure/maps/render/implementation/helpers/Utility.java

```diff
@@ -0,0 +1,56 @@
+package com.azure.maps.render.implementation.helpers;
+
+import java.util.Arrays;
+import java.util.List;
+
+import com.azure.core.annotation.ReturnValueWireType;
+import com.azure.core.http.rest.Response;
+import com.azure.core.http.rest.SimpleResponse;
+import com.azure.maps.render.implementation.models.BoundingBoxPrivate;
+import com.azure.maps.render.implementation.models.IncludeText;
+import com.azure.maps.render.implementation.models.MapTilesetPrivate;
+import com.azure.maps.render.models.BoundingBox;
+import com.azure.maps.render.models.Center;
+import com.azure.maps.render.models.MapTileset;
+
+public class Utility {
+    public static BoundingBoxPrivate toBoundingBoxPrivate(BoundingBox boundingBox) {
+        BoundingBoxPrivate boundingBoxPrivate = new BoundingBoxPrivate();
```

Comment:
This is not correct. BoundingBoxPrivate does not need this. It is generated code and should have setters.

The only time when this pattern is needed is when you want to convert from private to public, not the other way around. You can simply create a new BoundingBoxPrivate and call its setters.

Another tip on when to use the accessor pattern is that you want to modify the returned json, not what is going in.